wasatch/FeatureIdentificationDevice.py

```diff
@@ -1342,41 +1342,49 @@ def get_secondary_adc_raw(self) -> SpectrometerResponse:
     #        C3 = 8.78e-8
     # \endverbatim
     #
-    # @param raw    the value read from the thermistor's 12-bit ADC
-    def get_laser_temperature_degC(self, raw: float = None) -> SpectrometerResponse:
+    # @param raw    the value read from the thermistor's 12-bit ADC (should be 
```

Comment:
recommend converting this to a docstring by wrapping in 3 quotes and moving below the def line